hcfb-be/set_config.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+echo "Connect to local database or production database? input: (local or production)"
+
+read databaseMode
+
+if [ $databaseMode = local ]
```

Comment:
```suggestion
if [ $databaseMode = "local" ]
```

"local" is actually a bash keyword... better to be explicit that it's interpreted as a string.

Reply:
Fixed